guides/v2.2/cloud/project/project-multi-sites.md

```diff
@@ -8,157 +8,310 @@ functional_areas:
 
-    "http://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
+After successfully creating and testing the local installation to use multiple stores, you must prepare your Integration environment:
 
-    "https://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-    ```
+1. **Configure routes or locations**—specify how incoming URLs are handled by {{site.data.var.ee}}
+   -  [Routes for separate domains](#routes)
+   -  [Locations for shared domains](#locations)
+1. **Set up websites, stores, and store views**—configure using the {{site.data.var.ee}} Admin UI
+1. **Modify Magento variables**—specify the values of the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables in the `magento-vars.php` file
+1. **Deploy and test environments**—deploy and test the `integration` branch
 
-1.  Save your changes to the `routes.yaml` file.
+### Configure routes for separate domains {#routes}
 
-### Set up websites, stores, and store views
+Routes define how to process incoming URLs. Multiple stores with unique domains requires you to define each domain in the `routes.yaml` file. The way you configure routes depends on how you want your site to operate.
 
-In the Admin panel, set up your {{site.data.var.ee}} websites, stores, and store views. See [Set up multiple websites, stores, and store views in the Admin]({{ page.baseurl }}/config-guide/multi-site/ms_websites.html).
+{:.bs-callout-info}
+For Pro, you must create a [Support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to set up routes in the Staging or Production environment.
 
-It is important to name your websites, stores, and store views in your Cloud Admin the same as you did when you set up your local installation.
+#### To configure routes in an integration environment:
 
-### Modify the `magento-vars.php` file
+1. On your local workstation, open the `.magento/routes.yaml` file in a text editor.
+
+1. Define the domain and subdomains. The `mymagento` upstream value is the same value as the name property in the `.magento.app.yaml` file.
+
+   ```yaml
+   "http://{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "http://<second-site>.{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+   ```
+
+1. Save your changes to the `routes.yaml` file.
+
+1. Continue to the [_Set up websites, stores, and store views_ section](#set-stores).
+
+### Configure locations for shared domains {#locations}
+
+Where the routes configuration defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allows more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share the same domain.
```

Comment:
```suggestion
Where the routes configuration defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allow more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share the same domain.
```

Reply:
I saw this after the test run, but yes, this is tricky. Locations is a web property category. The brain can get confused here. I'd like to find a way to get rid of Allow. I don't like using that word, in fact. Maybe provides.... ugh, staring too long...